sidebars.js

```diff
@@ -52,6 +52,9 @@ module.exports = {
         "guides/update-guide/110-to-120",
         "guides/update-guide/100-to-110",
         "guides/update-guide/026-to-100",
+        {
+          Elasticsearch: ["guides/update-guide/elasticsearch/7-to-8"],
```

Comment:
🔧 We may want to move this block up right after Connectors and before the main Update Guides
<img width="321" alt="Screenshot 2023-04-05 at 17 07 22" src="https://user-images.githubusercontent.com/3511026/230123493-2eb657e6-4a39-481e-9264-d4172407ed68.png">

Reply:
I agree here, I'm afraid it could get lost in the list.